aten/src/TH/THBlas.cpp

```diff
@@ -2,3 +2,6 @@
 
 #include <TH/generic/THBlas.cpp>
 #include <TH/THGenerateAllTypes.h>
+
+#include <TH/generic/THBlas.cpp>
```

Comment:
I don't buy that we want to enable all these math functions on Half.  If you only need axpy because you are doing coalesce, only enable that one by modifying `TH/generic/THBlas.cpp` with `#if defined`.

Reply:
thought I needed them for something but I'll check again